10_install_server/12_install_root_server_rhel.txt

```diff
@@ -0,0 +1,99 @@
+=== Install Rudder Root server on Enterprise Linux ( RedHat or RedHat-like )
+
+==== Add the Rudder packages repository
+
+Run the following command as root:
+
+Note: <version> is your RHEL version
```

Comment:
I'm really not keen on leaving variables in these text blocks. Users like to copy/paste without editing commands.

Please either revert to having two options, or just put in RHEL_6 by default, and add a note for RHEL 7 users to change it if necessary.